packages/react-components/src/entities/EventSidebar/details/Summaries.js

```diff
@@ -98,21 +98,33 @@ export function Summaries({ event, setActiveEvent, addToSearch, addEventTypeToSe
     const mofHierarchyJoined = data?.mofResults.facet.eventTypeHierarchyJoined.sort(function (a, b) {
       return a.key.length - b.key.length
     });
+    //Calculate how may measurements attached to 'Survey','Sample','Find' etc
+    const measurements = mofHierarchyJoined.reduce(function(measurementsCount, record){
+      let total = record.events?.facet?.measurementOrFactTypes.reduce(function(count, mft){
```

Comment:
this isnt giving a total count of measurementOrFact entries, but a distinct list of measurementOrFactTypes

Reply:
@djtfmartin Its query has been modified, and it returns the results which contains the number of each  measurementOrFactType attached to the hierachyKey

`
  
     "key": "Survey / SiteVisit / Sample / Find",
            "count": 27,
            "events": {
              "facet": {
                "measurementOrFactTypes": [
                  {
                    "key": "biomass",
                    "count": 27
                  },
                  {
                    "key": "sizeClass",
                    "count": 27
                  }
                ]
              }
            }
          }
`

See this